src/Migrations/M022UpdatePostRevisions.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+
+/**
+ * Update the limit of post revisions.
+ */
+class M022UpdatePostRevisions extends MigrationScript
+{
+    /**
+     * Perform the actual migration.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     */
+    protected static function execute(MigrationRecord $record): void
+    {
+        global $wpdb;
+
+        // Check if exist
+        $MAX_REVISIONS = get_option('revisions_to_keep');
+        if (false === get_option('revisions_to_keep')) {
```

Comment:
The `not`(!) operator will be good fit for this `if` condition.(It covers both cases, the empty or false value)
```
if (! $MAX_REVISIONS) { 
...
}
```